main.cpp

```diff
@@ -52,84 +54,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:modWheel
+//  action 1: increases the pitch of the sound.
+modWheel.increasePitch();    
+//  action 2: Decreases the pitch of the sound.    
+modWheel.decreasePitch();
+//  action 3: scrolls through the different modes. 
+modWheel.scrollModes();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:arpeggiator       
+//  action 1:plays sequence of notes.
+arpeggiator.playSequence();
+//  action 2:opens arpeggiator menu
+arpeggiator.openMenu();
+//  action 3:sets note sequence
+arpeggiator.setSequence();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:key
+//  action 1:triggers a note to play
+key.triggerNote();
+//  action 2:tells a note to stop
+key.stopNote();
+//  action 3:sets the pitch of a note
+key.setPitch();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:mouse
+//  action 1:selects an item
+mouse.selectItem();
+//  action 2:right clicks an item
+mouse.rightClick();
+//  action 3:scrolls through items
+mouse.scrollItems();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:knob
+//  action 1:sets a value
+knob.setValue();
+//  action 2:increase value
+knob.increaseValue();
+//  action 3:decrease value
+knob.decreaseValue();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:eyes
+//  action 1:the eyes open    
+eyes.open();
+//  action 2: the eyes close
+eyes.close();
+//  action 3: the eyes see
+eyes.see();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:mouth
+//  action 1:the mouth speaks
+mouth.speak();
+//  action 2:the mouth eats
+mouth.eat();
+//  action 3:the mouth smiles
+mouth.smile();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: speaker
+//  action 1:the speakers makes sound
+speaker.makeSound();
+//  action 2:the speaker consumes energery
+speaker.consumeEnergy();
+//  action 3:the speaker gets the party going.
+speaker.getPartyGoing();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.